### ISSUE TITLE
Forgetting to set default column value to zero causes "undefined method `>' for false:FalseClass error" errors

### ISSUE THREAD

**Issue by jberry** [kenn/active_flag#3]
I've got my model setup with:

  ```flag :ratings, [:g, :pg, :pg13, :r, :nc17]```

And with debug console, the object looks good to my eyes:

```
>> MyModel.ratings.pairs

=> {"G"=>:g, "Pg"=>:pg, "Pg13"=>:pg13, "R"=>:r, "Nc17"=>:nc17}
```

I tried to load this into form_with checkboxes using:

```<%= form.collection_check_boxes :ratings, MyModel.ratings.pairs, :last, :first  %>```

and it's throwing the following error:

```
NoMethodError: undefined method `>' for false:FalseClass
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag/definition.rb:45:in `block in to_array'
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag/definition.rb:45:in `each'
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag/definition.rb:45:in `map'
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag/definition.rb:45:in `to_array'
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag/definition.rb:41:in `to_value'
	from /user/.rvm/gems/ruby-2.5.3/gems/active_flag-1.4.0/lib/active_flag.rb:23:in `block in flag'
```

Diving further down the debug-hole, it appears read_attribute(column) returns nil, which causes integer to be nil when passed to to_value which is then calling to_array with integer still nil:
``
0> column
=> :ratings

0> read_attribute(column)
=> nil
``

**Comment by jberry** [comment]
It appears this is caused if you don't set the column default value to 0. Not sure if there's a way to throw a prettier/helpful error.

**Comment by kenn** [comment]
Fixed with #7
